Cargo.toml

```diff
@@ -1,70 +1,63 @@
 [package]
-name = "punchy"
-version = "0.1.0"
+authors     = ["The Fish Folk Game & Spicy Lobster Developers"]
 description = "A 2.5D side-scroller beatemup"
-authors = ["The Fish Folk Game & Spicy Lobster Developers"]
-license = "MIT OR Apache-2.0"
-edition = "2021"
+edition     = "2021"
+license     = "MIT OR Apache-2.0"
+name        = "punchy"
+version     = "0.1.0"
 
 [workspace]
-members = [
-    ".",
-    "macros"
-]
+members = [".", "macros"]
 
 [dependencies.bevy]
-version = "0.8"
 default-features = false
-features = [
-    "x11",
-    "png",
-    "filesystem_watcher",
-    "bevy_gilrs"
-]
+features         = ["x11", "png", "filesystem_watcher", "bevy_gilrs"]
+version          = "0.9"
 
 [dependencies]
 punchy_macros = { path = "./macros" }
 
-anyhow = "1.0.58"
-bevy-parallax = "0.2.0"
-bevy_egui = "0.15.0"
-egui_extras = "0.18.0"
-bevy_kira_audio = { version = "0.11.0", features = ["mp3"] }
-bevy_rapier2d = { version = "0.16.0", features = ["debug-render"] }
-iyes_loopless = { version = "0.8.0", features = ["states"] }
-serde = { version = "1.0.137", features = ["derive"] }
-serde_yaml = "0.9.2"
-thiserror = "1.0.31"
-structopt = "0.3.26"
-rand = "0.8.5"
-getrandom = { version = "0.2", features = ["js"] }
+anyhow          = "1.0.58"
+bevy-parallax   = "0.3.0"
+bevy_egui       = "0.18.0"
+bevy_kira_audio = { version = "0.13.0", features = ["mp3"] }
+bevy_rapier2d   = { version = "0.19.0", features = ["debug-render"] }
+egui_extras     = "0.20.0"
+getrandom       = { version = "0.2", features = ["js"] }
+iyes_loopless   = { version = "0.9.0", features = ["states"] }
+rand            = "0.8.5"
+serde           = { version = "1.0.137", features = ["derive"] }
+serde_yaml      = "0.9.2"
+structopt       = "0.3.26"
+thiserror       = "1.0.31"
```

Comment:
Just curious, how did you format this file?

I think kind of like it, if there's an automatic way to do it.

Reply:
Oh i apologize. that is my vs-code extensions `Even Better Toml` auto formatting my toml for me. I forget to disable it on other projects since some people hate formatters being run on their code

Reply:
If @odecay doesn't have a problem with it, I'm fine, it's no problem.

I was just curious what you used to format it, because I might want to use it. :)

Reply:
I'm okay with it for now, if it becomes a problem to maintain formatting without that specific formatter in the future we can always change it back.